src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/PodTemplateStepExecution.java

```diff
@@ -30,7 +33,12 @@
 
     private static final transient String NAME_FORMAT = "%s-%s";
 
-    private final PodTemplateStep step;
+    @Inject(optional=true)
+    @SuppressFBWarnings(value = "SE_TRANSIENT_FIELD_NOT_RESTORED", justification = "not needed on deserialization")
+    private final transient PodTemplateStep step;
+    private String cloudName;
```

Comment:
If this is all we needed from the `step`, perhaps this did not need to be a field at all. Just initialize `cloudName` in the constructor, and make it `final`.

BTW if you are introducing support for resuming these steps for the first time, you can just make the `serialVersionUID` be, say, `1`. Hereafter you need to be careful to maintain the SVUID, and also to consider the effect of deserializing and old instance—for example, new fields might be null and you need to handle that possibility.